tests/_system/test_onedim_ckd_vs_mono.py

```diff
@@ -0,0 +1,310 @@
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+import eradiate
+from eradiate import unit_registry as ureg, unit_context_kernel as uck
+
+eradiate_dir = os.environ["ERADIATE_DIR"]
+output_dir = os.path.join(eradiate_dir, "test_report", "generated")
+
+
+def ensure_output_dir(path):
+    if not os.path.isdir(path):
+        os.makedirs(path)
+
+
+def init_mono_experiment(wavelengths, spp, reflectance, zeniths):
+    """
+    Initialise the monochromatic experiment.
+    """
+    eradiate.set_mode("mono_double")
+
+    return eradiate.experiments.OneDimExperiment(
+        measures=[
+            eradiate.scenes.measure.MultiDistantMeasure.from_viewing_angles(
+                spectral_cfg=eradiate.scenes.measure.MeasureSpectralConfig.new(
+                    wavelengths=wavelengths
+                ),
+                zeniths=zeniths,
+                azimuths=0.0 * ureg.deg,
+                spp=spp,
+            )
+        ],
+        illumination=eradiate.scenes.illumination.DirectionalIllumination(
+            zenith=50.0 * ureg.deg, azimuth=0.0 * ureg.deg
+        ),
+        atmosphere=eradiate.scenes.atmosphere.MolecularAtmosphere.afgl_1986(
+            id="molecules",
+            has_absorption=True,
+            has_scattering=True,
+            absorption_data_sets={},
+        ),
+        surface=eradiate.scenes.surface.LambertianSurface(
+            reflectance=reflectance,
+        ),
+    )
+
+
+def init_ckd_experiment(bin_set, bins, spp, reflectance, zeniths):
+    """
+    Initialise CKD experiment.
+    """
+    eradiate.set_mode("ckd_double")
+
+    return eradiate.experiments.OneDimExperiment(
+        measures=[
+            eradiate.scenes.measure.MultiDistantMeasure.from_viewing_angles(
+                spectral_cfg=eradiate.scenes.measure.MeasureSpectralConfig.new(
+                    bin_set=bin_set,
+                    bins=bins,
+                ),
+                zeniths=zeniths,
+                azimuths=0.0 * ureg.deg,
+                spp=spp,
+            )
+        ],
+        illumination=eradiate.scenes.illumination.DirectionalIllumination(
+            zenith=50.0 * ureg.deg, azimuth=0.0 * ureg.deg
+        ),
+        atmosphere=eradiate.scenes.atmosphere.MolecularAtmosphere.afgl_1986(
+            id="molecules",
+            has_absorption=True,
+            has_scattering=True,
+        ),
+        surface=eradiate.scenes.surface.LambertianSurface(
+            reflectance=reflectance,
+        ),
+    )
+
+
+def make_figure(ckd_results, mono_results_averaged, fname_plot, wavelength_bin):
+    fig = plt.figure(figsize=(8, 5))
+    brf_mono_averaged = mono_results_averaged.brf.squeeze()
+    brf_ckd = ckd_results.brf.squeeze()
+    brf_mono_averaged.plot(x="vza", ls="dotted", marker=".")
+    brf_ckd.plot(x="vza", ls="dotted", marker=".")
+
+    # Compute relative absolute difference and annotate plot with min, max and
+    # mean values
```

Comment:
I find the "relative absolute difference" nomenclature to be quite confusing in our context ("absolute bias" is actually a signed quantity in usual EO terminology). However, since this is correct vocabulary, let's clarify this:
```suggestion
    # Compute relative absolute difference and annotate plot with min, max and
    # mean values
    # NB: RAD(x, y) = abs(x - y) / x
```
Please also add similar detail in the text docstring.

Reply:
I understand your confusion. I derived the definition of "relative absolute difference" from the "mean absolute difference" definition from [wikipedia](https://en.wikipedia.org/wiki/Mean_absolute_difference#Relative_mean_absolute_difference) and it sounds a little weird as is 😃.
Where did you find the definition of absolute bias?
It looks like it would be nice to settle this vocabulary throughout eradiate since we use it a lot, don't you think?
As defined, the "relative absolute difference" can be negative, when `x` is negative (unlikely given that the quantities that we usually compare are defined positive).